proto/ots/bettor.proto

```diff
@@ -21,6 +21,44 @@ message BettorsResponse {
   }
 }
 
+message BettorUpsertRequest {
+  message Labels {
+    repeated string values = 1;
+  }
+
+  // Unique bettor id from operator’s system.
+  string id = 1;
+
+  //  Parameters bos and over_ask_factor has this specific constraints.
```

Comment:
Description is OK, but typically instead of `parameter` we typically use `property`